CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 
 * `azure` host plugin: Support multiple MSI identities
   ([PR](https://github.com/hashicorp/go-kms-wrapping/pull/97)
+* sessions: The default connect limit for new sessions is now unlimited. This avoids
+  the confusing behavior where a session is terminated as soon as a single connection
```

Comment:
I don't think this is the right way to put it. It's only confusing behavior to people that either don't look at the created target or don't understand what it means...and phrasing it this way might discourage people from doing anything other than `-1` even though in many cases this is indeed what you want. In fact, we've had users confused the other way -- hey, I set my connection count high and I killed my client and now the session shows as still open, does this mean anyone can now access my system, doesn't Boundary close sessions?

Reply:
How should we phrase this?